### PR TITLE
turkish translation added

### PR DESCRIPTION
**Pull request by erso44**
turkish translation added to .json file

### CODE REVIEW

**Review by spmse — CHANGES_REQUESTED** [review]
Please add another `.svg` file that shows the turkish flag, and add turkish to the `LocaleChanger.vue` component 👍

**Comment by erso44** [comment]
task finished.

File: src/components/LocaleChanger.vue [hunk 1]
```diff
@@ -29,6 +29,8 @@ export default {
           return require("../assets/lang_germany.svg");
         case "da":
           return require("../assets/lang_denmark.svg");
+        case "tr":
```
Comment (spmse) [inline src/components/LocaleChanger.vue:32]: In order to make turkish available in the selection, you need to add it to the components `data` (line 23).

**Review by spmse — CHANGES_REQUESTED** [review]

File: src/components/LocaleChanger.vue (hunk 1 above)
Comment (erso44) [reply src/components/LocaleChanger.vue:32]: @tlahmann  currently the flag is to big and vue does not scale it automatically. The flag .svg is added already. Can you re-size it ?

File: src/assets/lang_turkish.svg [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<svg xmlns="http://www.w3.org/2000/svg" width="1200" height="800">
```
Comment (tlahmann) [inline src/assets/lang_turkish.svg]: please remove `width` and `height` and add `viewBox="0 0 1200 800"` in this tag. This also fixes the [resize bug](https://github.com/ProjektKuro/kuro-web/pull/46#discussion_r403687530)

**Review by tlahmann — CHANGES_REQUESTED** [review]

**Pull request merged**